packages/charting/src/components/Legends/Legends.base.tsx

```diff
@@ -29,6 +29,7 @@ export interface ILegendState {
 }
 export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
   private _classNames: IProcessedStyleSet<ILegendsStyles>;
+  private centerLegends: boolean;
```

Comment:
It looks like this is only used in one method, so this private variable is unnecessary (there's no need to save the value of `this.props.centerLegends`).